frontend/src/modules/Pvt/settings/settings.tsx

```diff
@@ -0,0 +1,203 @@
+import React from "react";
+
+import { EnsembleIdent } from "@framework/EnsembleIdent";
+import { ModuleSettingsProps } from "@framework/Module";
+import { useEnsembleRealizationFilterFunc, useEnsembleSet } from "@framework/WorkbenchSession";
+import { MultiEnsembleSelect } from "@framework/components/MultiEnsembleSelect";
+import { CollapsibleGroup } from "@lib/components/CollapsibleGroup";
+import { Dropdown } from "@lib/components/Dropdown";
+import { PendingWrapper } from "@lib/components/PendingWrapper";
+import { RadioGroup } from "@lib/components/RadioGroup";
+import { Select, SelectOption } from "@lib/components/Select";
+
+import { useAtom, useAtomValue } from "jotai";
+
+import {
+    userSelectedEnsembleIdentsAtom,
+    userSelectedPvtNumsAtom,
+    userSelectedRealizationsAtom,
+} from "./atoms/baseAtoms";
+import {
+    pvtDataAccessorAtom,
+    selectedEnsembleIdentsAtom,
+    selectedPvtNumsAtom,
+    selectedRealizationsAtom,
+} from "./atoms/derivedAtoms";
+import { pvtDataQueriesAtom } from "./atoms/queryAtoms";
+
+import { DependentVariableSelector } from "../components/DependentVariableSelector/dependentVariableSelector";
+import { Interface, State } from "../state";
+import {
+    ColorBy,
+    PRESSURE_DEPENDENT_VARIABLE_TO_DISPLAY_NAME,
+    PhaseType,
+    PressureDependentVariable,
+} from "../typesAndEnums";
+import { computeRealizationsIntersection } from "../utils/realizationsIntersection";
+
+export function Settings({ settingsContext, workbenchSession }: ModuleSettingsProps<State, Interface>) {
+    const ensembleSet = useEnsembleSet(workbenchSession);
+    const filterEnsembleRealizationsFunc = useEnsembleRealizationFilterFunc(workbenchSession);
+
+    const selectedEnsembleIdents = useAtomValue(selectedEnsembleIdentsAtom);
+    const [, setSelectedEnsembleIdents] = useAtom(userSelectedEnsembleIdentsAtom);
```

Comment:
When updating atomValues only, use `useSetAtom`?

Reply:
Absolutely! I don't know what happened, but when I wrote the code, I did actually expect a hook like `useSetAtom` but did not get any suggestions from IntelliSense for import, so I just thought it does not exist. Double-checked now and changed the respective lines. Thanks for the push.